pkg/ping/build.go

```diff
@@ -3,16 +3,42 @@ package ping
 import (
 	"strings"
 
+	"github.com/pluralsh/deployment-operator/pkg/scraper"
+
 	console "github.com/pluralsh/console/go/client"
 	"github.com/samber/lo"
 	"k8s.io/apimachinery/pkg/version"
 )
 
 func pingAttributes(info *version.Info, pods []string, minKubeletVersion *string) console.ClusterPing {
 	vs := strings.Split(info.GitVersion, "-")
+	var insightComponents []*console.ClusterInsightComponentAttributes
+	if !scraper.AiInsightComponents.IsEmpty() {
+		insightComponents = make([]*console.ClusterInsightComponentAttributes, 0)
```

Comment:
minor things, if we send an empty list, it'll actually wipe the components on the server, so if empty just don't add to the cluster ping request at all (I'm hoping it has omitempty on it).  Also i think the full insight list should be fresh or not fresh, not individual ones, that way you can control easily whether to add the whole thing to the ping request.